### ISSUE TITLE
Adding Spotify uri 

### ISSUE THREAD

**Issue by danidfm** [mp911de/atsoundtrack#12]
I started to use this awesome plugin to my personal projects, I find incredibly funny and curious to know which music I was listening while I was developing my classes!

However, I think could be something useful (at least for me) to add the uri of the soundtrack to the comment. We have 2 variables (title and author), could be possible to add the uri link?

{title} - {author} - Play song: {uri} => Welcome to the jungle - Guns n' Roses - Play song: spotify:track:0bVtevEgtDIeRjCJbK3Lmv

Would be incredible if we even can add the functionality to put the URI as a link to spotify. I think there is somewhere in the Spotify API that allows the app to open spotify.

So, in summary, would be add the URI, so we can listening to that song just clicking on the comment on the class. What do you think?

I love the idea of have multiple devs working on my projects and listening the same songs that I listened while I originally develop the classes

**Comment by mp911de** [comment]
Spotify exposes the URL via AppleScript so it's possible to obtain it and add it to the mix. However, I'd argue to add the URL only as additional variable and leaving the default as is to retain simplicity. The config itself adds already some complexity.

We have #2 requesting the possibility to play tracks. Maybe it's worth to also add a parser/link handler to open Spotify with the URL if we discover one. In a future version, we could generate a playlist 😆 from a commit by inspecting which files/comments were introduced by a commit.

<img width="815" alt="screenshot 2017-11-29 08 58 53" src="https://user-images.githubusercontent.com/1035015/33364187-9eb029fe-d4e3-11e7-95da-cd64a68ee6ba.png">
